src/pages/SignUpPage/SignUpPage.jsx

```diff
@@ -0,0 +1,80 @@
+import React,{useReducer} from 'react'
+import {Nav} from '../../components/Nav/Nav';
```

Comment:
one suggestion instead of importing nav on every page import it in ```App.js```

Reply:
ok i will do this.